src/services/smv_storage.py

```diff
@@ -54,13 +57,75 @@ def upsert_verified_party(
         screen_name: str,
     ):
         now = datetime.utcnow().replace(tzinfo=timezone.utc)
+
+        # Check if it is not a transfer from one participant to the other
+        existing_parties_count = self.col_identities.count_documents(
+            {
+                "$or": [
+                    {"pub_key": pub_key},
```

Comment:
Does this mean we block duplicate pubkeys? If so wish should think about if we want to do that… I'm not sure if it should be a problem to change the pubkey I'm using 🤔 

Definitely only one reg per twitter account, and definitely should block registering the same pubkey to multiple accounts, but I am unsure about blocking people from changing their pubkey... cc @vegaprotocol/community

Reply:
You are still allowed to change your address/pubKey - it is covered with integration tests:
https://github.com/vegaprotocol/social-media-verification/pull/61/files#diff-744eebff736078f4500e76843681af173e5b6637e9ab8cd0740f702eaa8ad153R25